blobfuse2-gen1-artifcats.yaml

```diff
@@ -0,0 +1,189 @@
+            workingDirectory: $(WORK_DIR)
+          
+          # Publish the Artifacts
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: 'rustbin'
+            displayName: 'Publish Artifacts for adlsgen1fuse'
+            condition: succeeded()
+
+  # ---------------------- Build blobfuse and create a .deb package ------------------------------
+  - stage: BuildArtifact
+    dependsOn: BuildAdlsGen1Fuse
+    condition: succeeded('BuildAdlsGen1Fuse')
+    
+    jobs:
+      # Ubuntu builds
+      - job: BlobfuseBuild
+        strategy:
+          matrix:
+            Ubuntu-18:
+              container_image: 'ubuntu-18.04'
+              fuselib: 'libfuse-dev'
+              tags: 'fuse2'
+        
+        pool:
+          vmImage: $(container_image)
+
+        variables:
+          - group: NightlyBlobFuse
+          - name: root_dir
+            value: '$(System.DefaultWorkingDirectory)'
+          - name: work_dir
+            value: '$(System.DefaultWorkingDirectory)/azure-storage-fuse'
+        
+        steps:
+          - checkout: none
+          - script: |
+              git clone https://github.com/Azure/azure-storage-fuse
+            displayName: 'Checkout Code & List commits'
+            workingDirectory: $(root_dir)
+          
+          # list commits from past 12hrs
+          - script: |
+              git checkout `echo $(Build.SourceBranch) | cut -d "/" -f 1,2 --complement`
+              git --no-pager log --since="12 hours ago" --stat
+            displayName: 'List Commits'
+            workingDirectory: $(work_dir)
+          
+          # install dependencies required for compiling blobfuse
+          - script: |
+              sudo apt-get update --fix-missing
+              sudo apt-get install ruby-dev build-essential $(fuselib) -y
+              sudo gem install fpm -V
+            displayName: "Installing Dependencies"
+
+          # Copy adlsgen1fuse binary to blobfuse root
+          - task: DownloadBuildArtifacts@0
+            displayName: 'Download AdlsGen1Fuse Artifacts'
+            inputs:
+              artifactName: 'rustbin'
+              downloadPath: $(work_dir)
+
+          - script: |
+              cp ./rustbin/* ./
+              rm -rf ./rustbin/*
+              chmod 777 ./adlsgen1fuse
+              ls -l ./adlsgen1fuse
+            displayName: "Prepare for packaging"
+            workingDirectory: $(work_dir)
+          
+          # build blobfuse2 and generate binary
+          - template: 'azure-pipeline-templates/build-release.yml'
+            parameters:
+              work_dir: $(work_dir)
+              root_dir: $(root_dir)
+              tags: $(tags)
+
+          # place the generated binary files & any additional files in appropriate locations
+          - script: |
+              mkdir -p pkgDir/usr/bin/
+              mkdir -p pkgDir/usr/share/blobfuse2/
+              cp azure-storage-fuse/blobfuse2 pkgDir/usr/bin/blobfuse2
```

Comment:
Can we create a sample json file also and add it out repo as well as this package

Reply:
Added. Please check